content/articles/build-digital-clock-using-javascript/index.md

```diff
@@ -0,0 +1,309 @@
+let year = timeNow.getFullYear();
+let period = "AM";
+```
+
+> If we look closely, you will notice that we're using the ```timeNow``` variable to access the various methods in the JavaScript *date* class. We declared it up above in our function block.
+
+- Hours, minutes, and seconds are assigned to the ```hoursOfDay```, ```minutes```, and ```seconds``` variables.
+
+- The ```weekDay``` variable contains an array of days of the week.
+
+- The ```today``` variable gets a particular day from the ```weekDay``` collection.
+
+- The ```months``` variable gets the string representation of months of the year. We can use ```timeNow.getMonths``` to get months of the year but it is the number representation that will be printed using this method. Hence, the need to use the ```toLocaleString``` method that converts the month to string. The string *long* in the parenthesis means the complete spelling of the months should be printed. If you don't want that you can use *short* instead of long, and Feb instead of February will be printed for instance.
+
+- The ```year``` variable gets the year from the ```getFullYear()``` method.
+
+- The ```periodOfDay``` variable is initialized to *AM*, this prints the period of the day.
+
+I think that is enough explanation up there. Our next step would be to couple all of these variables in the function, here is how:
+
+```js
+function displayTime(){
+
+   const timeNow = new Date();
+
+   let hourOfDay = timeNow.getHours();
+   let minutes = timeNow.getMinutes();
+   let seconds = timeNow.getSeconds();
+   let weekDay = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
+   let today = weekDay[timeNow.getDay()];
+   let months = timeNow.toLocaleString("default", { month: "long" });
+   let year = timeNow.getFullYear();
+   let period = "AM"; 
+
+}
+```
+
+That is what our function looks like right now. But this is not enough to build our digital clock, the reason you're reading this article. The next step is to set the period of the day.
+
+Here is the code snippet to illustrate that:
+
+```js
+
+// Permit me to declare the period and hourOfDay again even though we've declared them before
+
+let periodOfDay = "AM";
+
+let hourOfDay = timeNow.getHours();
+
+// write a condition that set our period to AM or PM
+
+if(hourOfDay > 12) {
+   hourOfDay -= 12;
+   period = "PM";
+   /*This block checks whether the hour is greater than 12 and if that is true, 12 is subtracted from the hour. The result is then assigned back to ```hour``` and the period is set to PM. 12 is subtracted since we're building a 12-hour time clock*/
+}
+
+if(hourOfDay == 0) {
+   hourOfDay = 12;
+   period = "AM";
+  /*This block checks whether the hour is greater than 12 and if that is true, 12 is subtracted from the hour. The result is then assigned back to ```hour``` and the period is set to PM. 12 is subtracted since we're building a 12-hour time clock*/
```

Comment:
This comment is a duplicate of the comment on line 175. I assume you wanted to write something else in this comment to explain the condition for the period to be *AM*. Please fix this comment.